internal/sysfs/sysfs_test.go

```diff
@@ -686,47 +685,6 @@ func TestWriterAtOffset_Unsupported(t *testing.T) {
 	require.Equal(t, syscall.ENOSYS, err)
 }
 
-// Test_FileSync doesn't guarantee sync works because the operating system may
```

Comment:
we don't need to test this again because adapterFS only job is to return a platform.File, and the impl is already tested.